lib/order.rb

```diff
@@ -0,0 +1,55 @@
+class Order 
+  attr_reader :my_order 
+
+  def initialize
+    @my_order = []
+  end
+
+  def add_dish_to_order(name, quantity)
+    @name, @quantity = name, quantity
+    @my_order << { @name => @quantity }
+    return "#{quantity} x #{name} added to your order"
```

Comment:
It's usually better to not return "user-friendly" message strings as return values of a method that operates on data (again, to respect the SRP, but it's also easier to make mistakes if we'd need to verify/test this string message later). What you could do is return the dish that was added (or perhaps just `true` or `false` for a result, etc)